.github/workflows/tests.yml

```
@@ -160,22 +85,40 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.PAT }}
 
+      - name: Composer
+        if: steps.environment.outputs.has-php == '1'
```

actionlint (1.7.7)

property "environment" is not defined in object type {}

(expression)